community/modules/management/dependencies-installer/kubernetes_manifest/README.md

```diff
@@ -0,0 +1,40 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
Can you please add a description similar to [this](https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/main/modules/management/kubectl-apply/README.md#description).

Reply:
Tried to add a description but getting lint errors. I don't see a description in [kubectl](https://github.com/GoogleCloudPlatform/cluster-toolkit/tree/main/modules/management/kubectl-apply/kubectl as well. So not adding it.